integrations/opentelemetry/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id(libs.plugins.kotlin.jvm.get().pluginId)
+    id(libs.plugins.kotlinx.serialization.get().pluginId)
+    alias(libs.plugins.arrow.gradle.publish)
+    alias(libs.plugins.semver.gradle)
+    alias(libs.plugins.detekt)
+}
+
+dependencies { detektPlugins(project(":detekt-rules")) }
+
+repositories { mavenCentral() }
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+    toolchain { languageVersion = JavaLanguageVersion.of(11) }
+}
+
+detekt {
+    toolVersion = "1.23.1"
+    source = files("src/main/kotlin")
+    config.setFrom("../../config/detekt/detekt.yml")
+    autoCorrect = true
+}
+
+dependencies {
+    implementation(projects.xefCore)
+    implementation(libs.opentelemetry.api)
+    implementation(libs.opentelemetry.exporter.logging)
+    implementation(libs.opentelemetry.sdk)
+    implementation(libs.opentelemetry.semconv)
+    implementation(libs.opentelemetry.extension.kotlin)
+    implementation(libs.opentelemetry.exporter.otlp)
+
+    testImplementation(libs.junit.jupiter.api)
+    testImplementation(libs.kotest.property)
+    testImplementation(libs.kotest.framework)
+    testImplementation(libs.kotest.assertions)
+    testRuntimeOnly(libs.kotest.junit5)
+}
+
```

Comment:
I think you should add the spotless plugin here. Something like:

Inside `plugins`:
`alias(libs.plugins.spotless)`

And outside:

```gradle
spotless {
  kotlin {
    target("**/*.kt")
    ktfmt().googleStyle().configure {
      it.setRemoveUnusedImport(true)
    }
  }
}
```

Reply:
We should add spotless in a new PR